tools/package-tools/gulp/build-scss-pipeline.ts

```diff
@@ -0,0 +1,14 @@
+import {src} from 'gulp';
+import {join} from 'path';
+
+// These imports lack of type definitions.
+const gulpSass = require('gulp-sass');
+const gulpIf = require('gulp-if');
+const gulpCleanCss = require('gulp-clean-css');
```

Comment:
Looks like there are types for `gulp-sass` and `gulp-if`.

Reply:
There definitely is, but for such small things we didn't even consider installing the types

<sub>(no real benefit; just blows the package)</sub>

Reply:
That's not what the comment says though.

Reply:
Well, since we didn't install the types, the _imports_ basically lack of type definitions. Not the packages.

I see that the comment could be ambiguous, but we have similar comments across the whole tooling and it's mostly to indicate that we don't have types for these imports and to group them.